test/e2e/bots_test.go

```diff
@@ -176,6 +176,23 @@ func runBotTest(t *testing.T,
 	err = waitForDeploymentReady(deployNsCli, appCfg.Deployment.Name, appCfg.Deployment.WaitTimeout)
 	require.NoError(t, err)
 
+	cmdHeader := func(command string) string {
+		return fmt.Sprintf("`%s` on `%s`", command, appCfg.ClusterName)
+	}
+
+	t.Run("List actions stored in runtime config map", func(t *testing.T) {
```

Comment:
I'd remove a not necessary detail:
```suggestion
	t.Run("List actions", func(t *testing.T) {
```